validation/standard/standard-versions-sepolia.toml

```diff
@@ -5,6 +5,25 @@
 #   * proxied             : specify a standard "implementation_address"
 #   * neither             : specify neither a standard "address" nor "implementation_address"
 
+# Holocene https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.8.0-rc.3
+[releases."op-contracts/v1.8.0"]
+# Updated in this release
+system_config = { version = "2.3.0", implementation_address = "0x29d06Ed7105c7552EFD9f29f3e0d250e5df412CD" } # UPDATED IN THIS RELEASE
```

Comment:
This needs modifying since the implementation at this address actually has a version with a `-beta` suffix. We have a new deployment with the correct version, will update shortly.

Reply:
[be56fd8](https://github.com/ethereum-optimism/superchain-registry/pull/704/commits/be56fd8129f8cc919cb8915b3036788960ce266f)